courses/migrations/0002_auto_20151120_1503.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
```

Comment:
Can you rename this migration to something human readable, eg `0002_add_edx_author`